pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.apache</groupId>
     <artifactId>apache</artifactId>
-    <version>16</version>
+    <version>23</version>
```

Comment:
Just a question. Is there a reason why we need to upgrade to `23`?

Reply:
It is nice to keep everything in sync. The parent pom will also bump the plugins used by Maven.